modules/ROOT/pages/upgrade.adoc

```diff
@@ -83,6 +83,11 @@ $ kubectl get pods --all-namespaces
 ** xref:access-management-disclaimer.adoc[Configure a Disclaimer for Anypoint Private Cloud]
 ** xref:verify-nfs.adoc[NFS Server Prerequisites]
 
+[NOTE]
+When upgrading from a previous Anypoint Platform PCE version, all policies available in the previous version remain. Policies are not replaced, but additional policies might be included with the upgrade.
```

Comment:
"Policies are not replaced": isn't this redundant with "all policies in the previous version remain"? Or maybe I misunderstood. Does the paragraph mean "New versions of Anypoint Platform PCE always include the policies that are available in the previous version. However, new versions might add to the list of available policies."?